website/cue/reference/releases/0.43.0.cue

```diff
@@ -0,0 +1,447 @@
+				"""
+			contributors: ["esensar"]
+		},
+		{
+			type: "enhancement"
+			description: """
+				Add VECTOR_HOSTNAME env variable to override the hostname used in the Vector events and internal metrics.
+				This is useful when Vector is running on a system where the hostname is not meaningful, such as in a container (Kubernetes).
+				"""
+			contributors: ["nabokihms"]
+		},
+		{
+			type: "feat"
+			description: """
+				Add `update` support to the bulk API to Elasticsearch Sink.
+				"""
+			contributors: ["blackrez"]
+		},
+		{
+			type: "feat"
+			description: """
+				Introducing the first version of the [OpenTelemetry](https://opentelemetry.io/docs/what-is-opentelemetry/) sink.
+				"""
+			contributors: ["pront"]
+		},
+		{
+			type: "fix"
+			description: """
+				The request body of the Honeycomb sink should be encoded as an array according to [the API docs](https://docs.honeycomb.io/api/tag/Events#operation/createEvents).
+				"""
+			contributors: ["hgiasac"]
+		},
+		{
+			type: "enhancement"
+			description: """
+				Support compression for the Honeycomb sink. The `zstd` format is enabled by default.
+				"""
+			contributors: ["hgiasac"]
+		},
+		{
+			type: "enhancement"
+			description: """
+				Vector now supports two additional back-ends for loading secrets: `file`, for reading a set of
+				secrets from a JSON file, and `directory`, for loading secrets from a list of files.
+				"""
+			contributors: ["tie"]
+		},
+		{
+			type: "enhancement"
+			description: """
+				Add Gzip compression support to the `gcp_chronicle_unstructured` sink. See the [documentation](https://cloud.google.com/chronicle/docs/reference/ingestion-api#frequently_asked_questions).
+				"""
+			contributors: ["chocpanda"]
+		},
+		{
+			type: "enhancement"
+			description: """
+				The `sample` transform now has a `sample_rate_key` configuration option, which default to `sample_rate`. It allows configuring which key is used to attach the sample rate to sampled events. If set to an empty string, the sample rate will not be attached to sampled events.
+				"""
+			contributors: ["dekelpilli"]
+		},
+		{
+			type: "chore"
+			description: """
+				The global `expire_metrics_secs` configuration option now defaults to `300s` rather than being
+				disabled. To preserve the old behavior, set to a negative value to disable expiration.
+				"""
+			contributors: ["jszwedko"]
+		},
+		{
+			type: "fix"
+			description: """
+				Fix bug in implementation of Datadog search syntax which causes queries based on attributes with boolean values to be ignored.
+				"""
+			contributors: ["ArunPiduguDD"]
+		},
+		{
+			type: "fix"
+			description: """
+				The `gelf` codec now correctly deserializes the subsecond portion of timestamps rather than dropping
+				them.
+				"""
+			contributors: ["jszwedko"]
+		},
+		{
+			type: "enhancement"
+			description: """
+				Support for watching config file changes by polling at certain interval.
```

Comment:
```suggestion
				Support for watching config file changes by polling at certain interval rather than relying on notifications. This can be enabled setting `--watch-config-method` to `poll` where the interval can be configured via `--watch-config-poll-interval-seconds`.
```